File: HotelManager-Frontend/src/components/discount/Discount.js
const Discount = ({discount}) => {
    const {id, code, percent, validFrom, validTo } = discount;

    return(
        <div className="discount" key={id}>
            <h2>Code:{code}</h2>
            <p>Percentage:{percent}</p>
            <p>Valid from:{validFrom}</p>
            <p>Valid to:{validTo}</p>
        </div>
    );
};

File: HotelManager/HotelManager.Model/Discount.cs
using HotelManager.Model.Common;
using System;

namespace HotelManager.Model
{
    public class Discount : IDiscount
    {
        public Guid Id { get; set; }
        public string Code { get; set; }
        public int Percent {  get; set; }
        public DateTime ValidFrom { get; set; }
        public DateTime ValidTo { get; set; }
        public Guid CreatedBy { get; set; }
        public Guid UpdatedBy { get; set; }
        public DateTime DateCreated { get; set; }
        public DateTime DateUpdated { get; set; }
        public bool IsActive { get; set; }
    }
}
